dcos/dcos-ui
# feat(ClusterDropdown): authenticated cluster header label

> Be aware this PR does not contain the styles for the header (@tnguyen0 has been working on it).

Commit adds authenticated cluster dropdown label mechanism
to display the correct info when metadata available to the user
or to display the hardcoded string.

Closes DCOS-39679

## Testing

1. `npm start`
2. Go to `HeaderBar.tsx` comment out line 12 - 14.
3. Check out corresponding branch on privates plugin

**Case 1:**
User has admin permissions (access to metadata): user should see the cluster name and IP on the header.
![jul-31-2018 16-41-41](https://user-images.githubusercontent.com/549394/43466829-b7894560-94e0-11e8-9e40-efe82a2b9bf6.gif)


**Case 2**:
User doesn't have access to metadata = Cluster dropdown on Header displays `cluster` string only.
![jul-31-2018 16-40-16](https://user-images.githubusercontent.com/549394/43466822-b198c93c-94e0-11e8-884e-7b7dc3e8b11a.gif)

**How to create a user without metadata access**
![jul-31-2018 17-55-40](https://user-images.githubusercontent.com/549394/43471382-25d3d760-94eb-11e8-9433-af7fdad36ef9.gif)
